examples/external-mysql/run-tests.sh

```diff
@@ -12,31 +12,35 @@ source "${SCRIPTPATH}/../../.github/scripts/parse-versions.sh"
 # shellcheck source=/dev/null
 source "${TESTDIR}/common.sh"
 
-helm_install=(helm upgrade --install --create-namespace)
-ns=spire-system
-
 teardown() {
-  helm uninstall --namespace "${ns}" spire 2>/dev/null || true
-  kubectl delete ns "${ns}" 2>/dev/null || true
-
+  helm uninstall --namespace "spire-server" spire 2>/dev/null || true
   helm uninstall --namespace mysql mysql 2>/dev/null || true
-  kubectl delete ns mysql 2>/dev/null || true
+  kubectl delete ns spire-server 2>/dev/null || true
+  kubectl delete ns spire-system 2>/dev/null || true
 }
 
 trap 'trap - SIGTERM && teardown' SIGINT SIGTERM EXIT
 
-"${helm_install[@]}" mysql mysql --version "$VERSION_MYSQL" --repo "$HELM_REPO_MYSQL" \
-  --namespace mysql \
+kubectl create namespace spire-system --dry-run=client -o yaml | kubectl apply -f -
```

Comment:
Why not just `kubectl create namespace spire-system`?

Reply:
Idempotent. If it exists, it leaves it alone. If it doesn't, it creates it.

Reply:
`kubectl create` is not idempotent?

Reply:
No. it fails if it already exists. :/

```
$ kubectl create namespace foo
namespace/foo created
$ kubectl create namespace foo
Error from server (AlreadyExists): namespaces "foo" already exists
$ echo $?
1
```

create to apply is a common pattern to work around it.

Reply:
Got it, thanks.